README.md

```diff
@@ -470,7 +408,6 @@ Contributions are welcome. Please take a quick look at the [contribution guideli
  * [Water](https://github.com/shootingfly/water) - A library for writing HTML in plain Crystal
 
 ## Testing
- * [crotest](https://github.com/emancu/crotest) - A tiny and simple test framework
```

Comment:
I don't understand the criteria.
The shard is feature complete and stable, why remove it?

Reply:
See https://github.com/veelenga/awesome-crystal/pull/457#issuecomment-552223501

Reply:
Ok. I added weekly builds, and updated to crystal 0.33!